docs/pages/index.mdx

```diff
@@ -57,9 +63,223 @@ endpoints, infrastructure assets, and AI agents.
+          description: 'Replace long-lived secrets with identity-based auth',
+          href: './machine-workload-identity/'
+        },
+        {
+          title: 'Deploy CI/CD Pipelines',
+          description: 'Replace long-lived secrets in CI/CD pipelines',
+          href: './machine-workload-identity/machine-id/deployment/#cicd'
+        },
+        {
+          title: 'Secure Infrastructure as Code',
+          description: 'Manage IaC workflows in Terraform and Pulumi',
+          href: './admin-guides/infrastructure-as-code/terraform-provider/'
+        },
+        {
+          title: 'Hybrid & Multi-Cloud Authentication',
+          description: 'Universal identities across cloud platforms',
+          href: './machine-workload-identity/machine-id/deployment/'
+        },
+        {
+          title: 'Workload to Workload Authentication',
+          description: 'Service-to-service auth with mTLS',
+          href: './machine-workload-identity/workload-identity/spiffe/'
+        },
+        {
+          title: 'Identity Management for AI Agents',
+          description: 'RBAC for autonomous agents and processes',
+          href: './machine-workload-identity/workload-identity/introduction/'
+        }
+      ]
+    },
+    {
+      id: 'identity-governance',
+      title: 'Identity Governance',
+      href: './identity-governance/',
+      description: 'Manage identities by enforcing principles of least privilege and zero trust',
+      iconColor: '#EEEAFA',
+      iconComponent: IGSvg,
+      features: [
+        {
+          title: 'Request Temporary Elevated Access',
+          description: 'Eliminate standing privileges w/ just-in-time access',
+          href: './identity-governance/access-requests/'
+        },
+        {
+          title: 'Manage Standing Access for Teams',
+          description: 'Sync IdP groups to roles w/ automated reviews',
+          href: './identity-governance/okta/app-and-group-sync/'
+        },
+        {
+          title: 'Require Managed Devices for Access',
+          description: 'Guarantee access only from trusted devices',
+          href: './identity-governance/device-trust/'
+        },
+        {
+          title: 'Instantly Lock Identities & Sessions',
+          description: 'Lock compromised users and resources',
+          href: './identity-governance/locking/'
+        },
+        {
+          title: 'Integrate w/your Identity Provider(s)',
+          description: 'Okta, Entra ID, and Sailpoint w/SCIM group sync',
+          href: './identity-governance/okta/'
+        },
+        {
+          title: 'Federate Users to External Services',
+          description: 'Use Teleport as SAML IdP to 3rd-party apps',
+          href: './admin-guides/access-controls/idps/'
+        },
+        {
+          title: 'Monitor for Risky Access',
+          description: 'Identify risky access patterns and behaviors',
+          href: './identity-governance/access-monitoring/'
+        }
```

Comment:
I'd probably remove this one, for 2 reasons.

1. We aren't investing much engineering effort into Access Monitoring. It didn't really take off.
2. There's some overlap with Identity Security, where we're using AI to identify suspicious activity.